Add basic /cves/:cveName/exposed_clusters api endpoint

blayson opened the pull request:
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

jdobes commented on manager/controllers/cves/exposed.go:
```diff
@@ -0,0 +1,101 @@
+}
+
+func (c *Controller) BuildExposedClustersQuery(cveName string) *gorm.DB {
+	return c.Conn.Table("cluster").
```
Query doesn't filter by account, it shouldn't return all clusters in DB.

blayson commented:
I have changed `generate_swagger.sh` script to follow https://github.com/koalaman/shellcheck/wiki/SC2181
Added swagger documentation and change `db query` filter by `account ID`
